lib/bundler.js

```diff
@@ -38,10 +41,19 @@ module.exports = class ElementBundler extends Plugin {
 		return bundler.generateManifest([entry])
 			.then((manifest) => bundler.bundle(manifest)).then((result) => {
 				const html = parse5.serialize(result.documents.get(entry).ast);
+				const content = this.processString(html);
 
-				return fs.outputFile(outpath, html);
+				return fs.outputFile(outpath, content);
 			}).catch((err) => {
 				throw new Error(`ember-cli-polymer-bundler bundler.bundle() failure: ${err}`);
 			});
 	}
+
+	processString(input) {
+		if (this.autoprefixer.enabled) {
```

Comment:
You are not testing this. So it should reduce the coverage, but it isn't. I checked and it is broken. Opened a pr with a fix: #10. Check it out.